src/CampaignAdmin.php

```diff
@@ -155,6 +155,35 @@ public function readCampaigns()
         return $response;
     }
 
+    protected function getPlaceholderGroups()
+    {
+        $groups = [];
+
+        if (class_exists('SilverStripe\CMS\Model\SiteTree')) {
+            $class = singleton('SilverStripe\CMS\Model\SiteTree');
+            $groups[] = [
+                'baseClass' => DataObject::getSchema()->baseDataClass('SilverStripe\CMS\Model\SiteTree'),
+                'singular' => $class->i18n_singular_name(),
+                'plural' => $class->i18n_plural_name(),
+                'noItemText' => 'Add from ' . $class->i18n_plural_name() . ' area',
+                'items' => []
+            ];
+        }
```

Comment:
I'm not comfortable with the idea of cross-module references like this...
Can these be added by an extension hook, rather than hardcoded in to campaign admin?
It could even be an array of classes in a config.

Either extension hook or config will be fine, both could be a best of both worlds.